zrml/asset-router/src/tests/mod.rs

```diff
@@ -41,7 +41,7 @@ mod custom_types;
 mod destroy;
 mod inspect;
 mod managed_destroy;
-mod multi_currency;
 mod multi_lockable_currency;
 mod multi_reservable_currency;
+mod multicurrency;
```

Comment:
Why did we rename this? Seems more appropriate to call it `multi_currency` given that you're calling it `MultiCurrency` instead of `Multicurrency`.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1256/commits/2134e4a76ac9fbd07107baa10895a44c552e6c03